openpype/hosts/blender/api/plugin.py

```diff
@@ -741,27 +741,41 @@ def _load_library_datablocks(
             datacol.foreach_set("use_fake_user", seq)
 
         if self.bl_types & BL_OUTLINER_TYPES:
-            # Override datablocks if needed
-            if do_override:
-                # Get datablocks to override, which have no user in the loaded datablocks (orphan at this point)
-                datablocks_to_override = {
+            # Get datablocks to override, which have
+            # no user in the loaded datablocks (orphan at this point)
+            datablocks_to_override = {
                     d
                     for d, users in bpy.data.user_map(
                         subset=datablocks
                     ).items()
                     if not users & datablocks
                 }
+            
+            # Try to get the right asset container from imported collections.
+            # TODO this whole outliner entity is a bad idea
+            # it must be refactored to deal correctly with nested
+            # outliner entities using user_map()
```

Comment:
Can it be refactored here then or do you think it's not relevant to this PR?

Reply:
It is a very big work which must be tested and shouldn't be integrated in a bugfix.